### PR TITLE
Feature/cypress sonar testing

### PR DESCRIPTION
**Pull request by oalyman**
add sonar cube testing for cypress

### CODE REVIEW

File: e2e-cypress/Jenkinsfile.template [hunk 1]
```diff
@@ -10,6 +10,7 @@ odsComponentPipeline(
   ]
 ) { context ->
   stageTest(context)
+  odsComponentStageScanWithSonar(context)
```
Comment (michaelsauter) [inline e2e-cypress/Jenkinsfile.template:13]: Should we reverse the order? I think it is probably not needed to run the tests before doing the analysis, right? And tests will likely take quite a bit longer ...

Comment (ManuelFeller) [reply e2e-cypress/Jenkinsfile.template:13]: Agree with the reversed order. Running them after the SonarQube scan would also encourage you to do e2e tests locally to be more sure that that stage does not fail...

Comment (clemensutschig) [reply e2e-cypress/Jenkinsfile.template:13]: does not work - for SQ you need binaries :(

Comment (michaelsauter) [reply e2e-cypress/Jenkinsfile.template:13]: @clemensutschig This is TypeScript ... no binaries. IIRC SQ scanning works anytime against the source files only.

**Comment by clemensutschig** [comment]
Not sure.. i had isses in the spock one.. but feel free to try.

Michael Sauter <notifications@github.com> schrieb am Mo., 3. Aug. 2020,
12:23:

**Pull request merged**